### PR TITLE
Requirement file

### PR DESCRIPTION
**Pull request by jacekszubert**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+click==6.7
+boto3==1.4.4
```
Comment (xaf) [inline requirements.txt]: `boto3>=1.4.2`

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+click==6.7
```
Comment (xaf) [inline requirements.txt]: `click>=6.6`

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+boto3==1.4.4
+pyOpenSSL==16.2.0
```
Comment (xaf) [inline requirements.txt]: `pyOpenSSL>=16.2.0`

**Review by xaf — CHANGES_REQUESTED** [review]
In addition to the inline comments, same as for pull request #3 - please change:
 - Rebase cleanly, your pull requests should not contain a "Merge pull request #1 from XaF/master" commit, but be based on the master of this repository
 - Add your signoff to your commits (git commit -s)
Once done, I'll merge this pull request.
Thanks a lot for your contribution :o)

**Comment by jacekszubert** [comment]
:+1:

**Review by xaf — APPROVED** [review]

**Comment by xaf** [comment]
Merged in commit 38e392cdc0b25ed91203a958203cffddaba13c48 with slight changes in version numbers
Thanks

**Pull request closed without merging**